sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -1077,7 +1077,7 @@ inline std::string GetConnectionKey(std::string const& host, CurlTransportOption
   {
     key.append("0");
   }
-  if (options.SSLOptions.NoRevoke)
+  if (!options.SSLOptions.EnableCertificateRevocationListCheck)
```

Comment:
Was this change necessary for correctness? I would imagine if someone set `EnableCRL = true`, that would mean you append  "1" to the key, but if CRL checking is off, you'd append "0".  Why negate it here?

Reply:
yes, it was either changing all unit tests from 1 to 0 or change the sentinel here.